src/tr1/game/lara/common.c

```diff
@@ -139,22 +192,71 @@ void Lara_Control(void)
             g_Lara.head_rot.y = 0;
             g_Lara.torso_rot.x = 0;
             g_Lara.torso_rot.y = 0;
+        } else {
+            g_Lara.water_status = LWS_WADE;
+            Item_SwitchToAnim(item, LA_STAND_IDLE, 0);
+            item->current_anim_state = LS_STOP;
+            item->goal_anim_state = LS_WADE;
+            Item_Animate(item);
+            item->fall_speed = 0;
         }
```

Comment:
Can we flip this condition so that we test for the positive wading scenario first?

Reply:
Done 👍